cli/pcluster/commands.py

```diff
@@ -486,7 +486,11 @@ def _set_asg_limits(asg_name, min, max, desired):
 def _get_asg_instances(stack):
     asg = boto3.client("autoscaling")
     asg_name = _get_asg_name(stack)
-    asg = asg.describe_auto_scaling_groups(AutoScalingGroupNames=[asg_name]).get("AutoScalingGroups")[0]
+    auto_scaling_groups = asg.describe_auto_scaling_groups(AutoScalingGroupNames=[asg_name]).get("AutoScalingGroups")
+    if not auto_scaling_groups:
+        LOGGER.error("Unable to retrieve ASG info. Please check cluster status.")
```

Comment:
Should we use LOGGER.critical ?

Reply:
I see critical to be more appropriate for unexpected exceptions/unhandled exceptions. In this case we know that probably the cluster is being deleted or failed to create and that is why we are recommending to check its status. What do you think?

Reply:
ok, I agree.